modules/undertoneBidAdapter.js

```diff
@@ -120,8 +122,20 @@ export const spec = {
         sizes: bidReq.sizes,
         params: bidReq.params
       };
+      const videoMediaType = deepAccess(bidReq, 'mediaTypes.video');
```

Comment:
this should potentially say &  deepAccess(bidReq, 'mediaTypes.video.context') != 'outstream or something of the sort

Reply:
@patmmccann i believe our team already reached out to you via email, however please reach out to pubams@undertone.com for further discussion/ next steps

Reply:
@oranperion this is a community issue, not just our account. What is the status of this? Should we revert this pr as it introduces the bug or are you planning a patch?

Reply:
@patmmccann issue already been solved on backend, there is already high fill rate on the placement id 2002001, and an email about this information been sent to you and your company last week, where a member of your company - Isaiah Ward <iward@cafemedia.com> conform issue resolved

Reply:
Thanks for looping in the community here, glad all is resolved!